src/Formik.tsx

```diff
@@ -499,11 +509,18 @@ export function useFormik<Values extends FormikValues = FormikValues>({
           value,
         },
       });
-      return validateOnChange && shouldValidate
-        ? debouncedValidateFormWithLowPriority(
+      if (validateOnChange && shouldValidate) {
+        if (value.length < 5 || value.endsWith(' ')) {
```

Comment:
Why 5? What a magic number? Let's make a constant to make this part a bit more clear.

Reply:
Or better yet, it should be configurable and not just number, but that space behavior. Ideally, some optional function that contains that logic by default, but can be overridden.